include/spirv-tools/optimizer.hpp

```diff
@@ -173,7 +173,7 @@ class Optimizer {
   // for |OptimizerOptions| to see which options can be set.
   bool Run(const uint32_t* original_binary, const size_t original_binary_size,
            std::vector<uint32_t>* optimized_binary,
-           const spv_optimizer_options opt_options) const;
+           const spv_optimizer_options& opt_options) const;
```

Comment:
`spv_optimizer_options` is just a pointer if you look at the typedefs.  No need for the reference.

Reply:
Ah, yes, sorry! Messed it up with `OptimizerOptions`. :(

Reply:
Fixed.